crates/zed/src/languages/clojure/config.toml

```diff
@@ -0,0 +1,10 @@
+name = "Clojure"
+path_suffixes = ["clj", "cljs"]
+line_comments = [";; "]
+autoclose_before = "}])"
+brackets = [
+    { start = "{", end = "}", close = true, newline = true },
+    { start = "[", end = "]", close = true, newline = true },
+    { start = "(", end = ")", close = true, newline = true },
+    { start = "\"", end = "\"", close = true, newline = false, not_in = ["string"] },
+]
```

Comment:
We must also specify `word_characters = ["-"]`, as that has significant impact on how navigating around source files in a given language works. E.g. without it "go to word end" when your cursor is at the start of `some-identifier` will go to `-`, not `r`.